login-workflow/src/new-architecture/screens/AccountDetailsScreen/AccountDetailsScreen.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { AccountDetailsScreenBase, AccountDetailsScreenProps } from '../AccountDetailsScreen';
+import { useLanguageLocale } from '../../hooks';
+import { useRegistrationWorkflowContext } from '../../contexts';
+
+type AccountDetailsFullScreenProps = AccountDetailsScreenProps & {
+    title?: string;
+    instructions?: string;
+};
+
+export const AccountDetailsScreen: React.FC<AccountDetailsFullScreenProps> = (props) => {
+    const { t } = useLanguageLocale();
+    const regWorkflow = useRegistrationWorkflowContext();
+    const { nextScreen, previousScreen, screenData } = regWorkflow;
+    const [firstName, setFirstName] = useState(screenData.AccountDetails.firstName ?? '');
```

Comment:
Please destructure to:
`const {firstName, lastName} = screenData.AccountDetails;`